streamlit_app.py

```diff
@@ -300,6 +304,14 @@ def draw_state_counties():
             .transform_filter(county_multiselect)
 
     # Draw maps side-by-side
+    map_background = alt.Chart(data=counties)\
+        .mark_geoshape(stroke='black', strokeWidth=1, fill='lightgray')\
+        .transform_calculate(state_id="(datum.id/1000)|0", FIPS="datum.id")\
+        .transform_filter((alt.datum.state_id == selected_state_fips) & (alt.datum.id % 1000 != 0))\
+        .properties(width=400, height=400)\
+        .encode(tooltip=[alt.Tooltip('id:N', title='FIPS')])
```

Comment:
I think if possible we should include the Location field here in the tooltip to keep it consistent with counties that do have data.